examples/binary_examples/PyFstat_example_binary_mcmc_vs_grid_comparison.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python
+
+import pyfstat
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Set to false to include eccentricity
+circular_orbit = False
+
+label = (
+    os.path.splitext(os.path.basename(__file__))[0] + "_circular_orbit"
+    if circular_orbit
+    else ""
```

Comment:
This dumps all files into "PyFstat_example_data" if `not circular_orbit`! Missing a pair of brackets.

Reply:
That's right, behold the dangers of the ternary operator.